vocode/streaming/output_device/exotel_output_device.py

```diff
@@ -0,0 +1,138 @@
+from __future__ import annotations
+
+import asyncio
+import base64
+import json
+from typing import List, Optional, Union
+
+from fastapi import WebSocket
+from fastapi.websockets import WebSocketState
+from loguru import logger
+from pydantic import BaseModel
+
+from vocode.streaming.output_device.abstract_output_device import AbstractOutputDevice
+from vocode.streaming.output_device.audio_chunk import AudioChunk, ChunkState
+from vocode.streaming.telephony.constants import DEFAULT_SAMPLING_RATE, EXOTEL_AUDIO_ENCODING
+from vocode.streaming.utils.create_task import asyncio_create_task
+from vocode.streaming.utils.dtmf_utils import DTMFToneGenerator, KeypadEntry
+from vocode.streaming.utils.worker import InterruptibleEvent
+
+
+class ChunkFinishedMarkMessage(BaseModel):
+    chunk_id: str
+
+
+MarkMessage = Union[ChunkFinishedMarkMessage]  # space for more mark messages
+
+
+class ExotelOutputDevice(AbstractOutputDevice):
```

Comment:
for this, could you subclass `TwilioOutputDevice` and add some configuration instead so it works? would love to not repeat so much code